mass_mailing_confirm/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Mass mailing confirm
+====================
+
+This module extends the functionality of mass_mailing addon adding a
+confirmation dialog after user click to "Send to all" button at Mass mailing
```

Comment:
are you sure?
"confirmation dialog after user click "Send to all" button in Mass mailing"